RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungbileAssetRow+Reducer.swift

```diff
@@ -131,6 +132,9 @@ extension NonFungibleAssetList {
 					state.isLoadingResources = false
 				}
 				return .none
+
+			case .refreshResources:
+				return loadResources(&state, pageIndex: 0)
```

Comment:
I think that `state.nextPageCursor` needs to also be reset to nil if pageIndex is set to zero. And more, shouldn't the `tokens` be reset?

I guess that this update works properly for a single page NFT collection.